### DISCUSSION TITLE
Could you make this compatible with the depth model?

### DISCUSSION THREAD

**Discussion by dnoparker** [TheLastBen/fast-stable-diffusion#1085] [Ideas]
https://huggingface.co/stabilityai/stable-diffusion-2-depth

It would be really cool if we could train the depth model! This guy has done it here -
https://github.com/epitaque/dreambooth_depth2img

Here is what is possible with the depth model -
https://twitter.com/szwajkakacper/status/1604164769224839168?s=46&t=gQC6A9W6qNcLWzuP8z5tTQ

I can see loads of use cases for a trained version.

**Comment by rycossette** [comment]
Totally, I would use this all the time

**Comment by TheLastBen** [comment]
I'll look into it
